docker-compose.ci.yml

```diff
@@ -1,33 +0,0 @@
-version: '3.2'
```

Comment:
I assume the file was removed because it's now kind of the same as docker-compose?